README.md

```diff
@@ -68,9 +74,51 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+This method should be called right before the split method because words with punctuation will not be evaluated correctly if it is not applied.
+
 
 3. Write a method called `happy_days` to determine how many logged entries it takes until there have been three :-) happy days.
 
+
+def happy_days(text)
+count = 0
+happy = 0
+sad = 0
+neutral = 0
+(0..text.count-1).map do |i|
+  days =  [text[i].split(" ")[0], analyze_mood(text[i])]
+  count += 1
+  if days[1].include? "("
+  sad += 1
+  elsif days[1].include? "|"
+  neutral += 1
+  else days[1].include? ")"
+  happy += 1
```

Comment:
Ooooh! This is interesting. I hadn't thought to do it that way. It definitely looks like it works, but I think it would have made for more readable code if you said `days[1] == ":-("` etc.

The body's of the if's need to be indented here.